### PR TITLE
Removing Save button from status change

### PR DESCRIPTION
**Pull request by hamalmar**

### CODE REVIEW

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 1]
```diff
@@ -58,17 +58,12 @@ angular.module('dashboard')
...
-                            else {
-                                $uibModalInstance.dismiss();
-                            }
+                        $scope.clicked = function () {
+                            $uibModalInstance.dismiss();
                         };
 
                         $scope.itemSelected = function (selectedItem) {
-                            $scope.selected = selectedItem;
+                            $uibModalInstance.close(selectedItem);
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.status.ctrl.js:61]: $scope.selected not used after this change, should remove its initialisation

File: app/scripts/controllers/meeting.status.ctrl.js [hunk 2]
```diff
@@ -58,17 +58,12 @@ angular.module('dashboard')
                             return angular.equals(item, $scope.selected);
```
Comment (tkenakka) [inline app/scripts/controllers/meeting.status.ctrl.js]: After $scope.selected is removed, whole use-case for `isSelected` disappears? It's dead code and should be removed from .js and .html?

**Comment by tkenakka** [comment]
Didn't test
LGTM but with comments

**Comment by hamalmar** [comment]
isSelected handling removed from code

**Pull request merged**